spec/stringProcessing/syllables/countFrenchSpec.js

```diff
@@ -0,0 +1,229 @@
+let countSyllableFunction = require( "../../../js/stringProcessing/syllables/count.js" );
+let forEach = require( "lodash/forEach" );
+
+/**
+ * Helper to tests syllable count.
+ *
+ * @param {array}  testCases List of cases to test.
+ * @param {string} locale    Locale to use.
+ *
+ * @returns {void}
+ */
+function testCountSyllables( testCases, locale ) {
```

Comment:
This same helper function is also used in `countDutchSpec.js`. It would be good to move it to a separate file (e.g., a file called `testCountSyllables.js` in the folder `spec/helpers`). This separate file can then be imported into both `countFrenchSpec.js` and `countDutchSpec.js`. That way, we avoid having a duplicate of the same code.

Reply:
We tried, but It is is not possible to have a separate helper file with the function